pages/stakeholder-and-user-interviews.md

```diff
@@ -30,3 +30,7 @@ To build consensus about the problem statement and research objectives.
 ## Applied in government research
 
 No PRA implications. The PRA explicitly exempts direct observation and non-standardized conversation, 5 CFR 1320.3(h)3. See the methods for [Recruiting](../recruiting/) and [Privacy](../privacy/) for more tips on taking input from the public.
+
+## Additional resources
+
+ - [&ldquo;Build empathy with stakeholder interviews, part 1: Preparation&rdquo; by Andrew Maier](https://18f.gsa.gov/2016/06/20/build-empathy-with-stakeholder-interviews-part-1-preparation/)
```

Comment:
Hi @andrewmaier – can you format the URL to match the precedent from our other references? Essentially, remove the quotation marks, end the blog title with a period, remove the 'by' in front of your name, and make it so that only the blog title is the link text. You can see an example here: https://methods.18f.gov/discover/heuristic-analysis/

Thanks!